libs/dal/package.json

```diff
@@ -20,6 +20,9 @@
   "license": "ISC",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "peerDependencies": {
+    "@nestjs/common": "10.2.2"
+  },
```

Comment:
🗒 **note (non-blocking):** dal used to be independent package :(

Reply:
The reason is that I need `@Inject` to change the implementation of the affected repositories e.g. `UserRepository` based on the EE flag while maintaining the original location of the UserRepository to avoid changing imports in 100s files:

```typescript
import { Inject } from '@nestjs/common';
import { Types } from 'mongoose';

export class UserRepository implements IUserRepository {
  constructor(@Inject('USER_REPOSITORY') private userRepository: IUserRepository) {}
```

I'm adding this to a TODO list and will think about ways to keep `@novu/dal` independent